### PR TITLE
arash/dmt5

### PR DESCRIPTION
**Pull request by arash-fs**
# Description

Summary of the change and/or issue that is fixed.

Changes:

-   fixes issue with item
-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/dmt5/_dmt5-video.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            ? 'color: var(--color-black-3); border-left: 5px solid var(--color-red)'
+            : 'opacity: 0.2; border-left: 5px solid rgb(0, 0, 0, 0)'};
+`
+const Step3 = styled(Header)`
+    ${StepCommon}
+    ${props =>
+        props.current_step_image === 'dmt-5-step-3.png'
+            ? 'color: var(--color-black-3); border-left: 5px solid var(--color-red)'
+            : 'opacity: 0.2; border-left: 5px solid rgb(0, 0, 0, 0)'};
+`
```
Comment (sara-fs) [inline src/pages/dmt5/_dmt5-video.js]: Why do we have 3 Step components here, when all are functioning the same?

File: src/components/elements/card.js [hunk 2]
```diff
@@ -109,24 +112,28 @@ export const Card = ({ children, Icon, title, content, width, min_height, is_inl
...
                             <IconContainer>
-                                <Icon />
-                                <Header as="h4" weight="500">
-                                    {title}
-                                </Header>
+                                <div>
+                                    <Icon />
+                                </div>
+                                <CardContentContainer>
+                                    <Header as="h4" weight="bold">
```
Comment (sara-fs) [inline src/components/elements/card.js]: The default `weight` for Header component is `bold`. 
`<Header as="h4">`

File: src/components/elements/card.js [hunk 3]
```diff
@@ -109,24 +112,28 @@ export const Card = ({ children, Icon, title, content, width, min_height, is_inl
...
-                                    {title}
-                                </Header>
-                                <Content content={content} />
-                            </ContentWrapper>
-                        </>
-                    )}
+                            <>
+                                <Icon />
+                                <ContentWrapper>
+                                    <Header as="h4" weight="bold">
```
Comment (sara-fs) [inline src/components/elements/card.js]: Same here.

File: src/pages/dmt5/_dmt5-video.js (hunk 1 above)
Comment (arash-fs) [reply src/pages/dmt5/_dmt5-video.js]: as I mentioned here https://github.com/binary-com/deriv-com/pull/365#discussion_r353593956 we can't use one component here

File: src/components/elements/card.js (hunk 2 above)
Comment (arash-fs) [reply src/components/elements/card.js]: will fix it in shared component :pray:

File: src/components/elements/card.js (hunk 3 above)
Comment (arash-fs) [reply src/components/elements/card.js]: will fix it in shared component :pray:

**Pull request closed without merging**